android/sdl_android/src/main/java/com/smartdevicelink/proxy/SdlProxyBase.java

```diff
@@ -4313,7 +4315,7 @@ public void sendSequentialRequests(final List<? extends RPCMessage> rpcs, final
 			throw new SdlException("You must send some RPCs", SdlExceptionCause.INVALID_ARGUMENT);
 		}
 
-		int requestCount = rpcs.size();
+		final int requestCount = rpcs.size();

```

Comment:
This name was changed in the `LifeCycleManager` version. We should maintain one or the other, your choice